tox.ini

```diff
@@ -81,6 +85,9 @@ commands =
 ; implicit Any due to unfollowed import would result).
   mypyinstalled: mypy --namespace-packages opentelemetry-api/tests/mypysmoke.py --strict
 
+[testenv:py34-test-opentracing-shim]
+commands =
+  python -m unittest discover --pattern 'test_*[!asyncio, !tornado].py'
```

Comment:
Please use `pytest` here to be consistent.

Reply:
Done.